amplipi/ctrl.py

```diff
@@ -335,6 +279,11 @@ def reinit(self, settings: models.AppSettings = models.AppSettings(), change_not
       if stream.id:
         try:
           self.streams[stream.id] = amplipi.streams.build_stream(stream, self._mock_streams)
+          # If we're in LMS mode, we need to start these clients on each boot, not when they get assigned to a
+          # particular source; the client+server connection bootstrapping takes a while, which is a less than ideal
+          # user experience.
+          if self.lms_mode and stream.type == 'lms':
+            self.streams[stream.id].activate()
```

Comment:
I considered doing this in the constructor but the controller is the bit that knows & cares about if it's in LMS mode.